src/vs/workbench/contrib/debug/browser/debugAdapterManager.ts

```diff
@@ -157,6 +157,11 @@ export class AdapterManager extends Disposable implements IAdapterManager {
 					},
 					'presentation': presentationSchema,
 					'internalConsoleOptions': INTERNAL_CONSOLE_OPTIONS_SCHEMA,
+					'multipleSessionWarning': {
```

Comment:
I'd prefer `suppressMultipleSessionWarning`

Reply:
`suppressMultipleSessionWarning` is OK, an alternative could be `suppressConcurrentSessionWarning`.